app/controllers/publish/providers_controller.rb

```diff
@@ -1,9 +1,31 @@
 module Publish
   class ProvidersController < PublishController
     rescue_from ActiveRecord::RecordNotFound, with: :render_not_found
+    decorates_assigned :provider
 
     def index
       authorize :provider, :index?
+
+      page = (params[:page] || 1).to_i
+      per_page = 10
+      @pagy, @providers = pagy(providers.order(:provider_name), page: page, items: per_page)
```

Comment:
ordered by provider_name, we have some names that start with a space....

Reply:
[["19F", " Bransgore Initial teacher Training (BITT) – School Direct"], ["1ES", " Harbour Learning Trust"], ["1GW", " Josie Thirkell Institute of Education"], ["1X3", " Linwood Training, Support and Advice"], ["1YD", " Nottingham Catholic Initial Teacher Training Partnership"], ["2CP", " SAND Training & Outreach"]]

Reply:
Yes